src/index.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+import './style.css';
+
+const todoList = [
+  {
+    index: '1',
+    description: 'cook my brakfast',
+  },

```

Comment:
- [optional ] your list structure works well.  Please kindly add boolean for completed in your array of task.
`completed : true /false.` 
this will help on your checkboxes